.circleci/config.yml

```diff
@@ -1,6 +1,20 @@
-version: 2
+version: 2.1
+
+workflows:
+  test_and_deploy:
```

Comment:
can you add an explanation for all these changes. i dont get what this change is for?

Reply:
aah so if i understand this right.. . this code snippet says the following

- there are two jobs `test` and `deploy`
- `test` runs always 
- `deploy` runs only on comment that have a tag `vXXX` which would be release tags.

As i said in the thread below, we want to do the releases manually for now because we are polishing our release process. So instead of changing this circleci configuration, can you put all the steps of the python release in a script that can be run manually?